ivy/array/array.py

```diff
@@ -689,7 +689,7 @@ def __rrshift__(self, other):
         --------
         >>> a = 32
         >>> b = ivy.array([0, 1, 2])
-        >>> y = a >> b
+        >>> y = b.__rrshift__(a)
```

Comment:
Check the [doc](https://lets-unify.ai/ivy/deep_dive/13_docstring_examples.html), we don't need to add examples with the special method called explicitly. This will already trigger the reverse example, as the integer `__rshift__` special method will not have a rule for dealing with `ivy.Array` instances, and so the `__rrshift__` method will be called on `b` (the `ivy.Array`) anyway when calling `a >> b`

Reply:
Sure I can revert it